tests/compilation/test_integration_further_SDKs.py

```diff
@@ -23,17 +23,24 @@
 from qiskit.transpiler.layout import TranspileLayout
 from qiskit.transpiler.passes import CheckMap, GatesInBasis
 
-from mqt.predictor.rl import helper
+import mqt.predictor.rl.actions
+import mqt.predictor.rl.parsing
```

Comment:
same comment as in the other test file. this just creates a lot of repetitive and long code in the actual tests.